sdk/communication/azure-communication-identity/README.md

```diff
@@ -135,28 +138,35 @@ communicationIdentityClient.revokeTokens(user);
 ### Deleting a user
 Use the `deleteUser` function to delete a user.
 
-<!-- embedme ./src/samples/java/com/azure/communication/identity/ReadmeSamples.java#L149-L150 -->
+<!-- embedme ./src/samples/java/com/azure/communication/identity/ReadmeSamples.java#L148-L149 -->
 ```java
 // delete a previously created user
 communicationIdentityClient.deleteUser(user);
 ```
 
-## Contributing
-
-This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
+## Troubleshooting
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+All user token service operations will throw an exception on failure.
+<!-- embedme ./src/samples/java/com/azure/communication/identity/ReadmeSamples.java#L157-L161 -->
+```java
+try {
+    CommunicationUserIdentifier user = communicationIdentityClient.createUser();
+} catch (RuntimeException ex) {
+    System.out.println(ex.getMessage());
+}
+```
 
-This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+## Next steps
 
+Check out other client libraries for Azure communication service
 
-## Troubleshooting
+## Contributing
 
-In progress.
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
 
-## Next steps
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
-Check out other client libraries for Azure communication service
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
```

Comment:
The below links still have `administration` in them and should use `identity` instead

Reply:
Thanks Dom I've fixed in latest push